tonicpow/includes/class-tonicpow.php

```diff
@@ -179,32 +188,32 @@ private function define_admin_hooks()
 	private function define_public_hooks()
 	{
 
-		$plugin_public = new Tonicpow_Public($this->get_plugin_name(), $this->get_version());
+		$plugin_public = new Tonicpow_Public( $this->get_plugin_name(), $this->get_version() );
 
-		$this->loader->add_action('wp_enqueue_scripts', $plugin_public, 'enqueue_styles');
-		$this->loader->add_action('wp_enqueue_scripts', $plugin_public, 'enqueue_scripts');
-		$this->loader->add_action('init', $plugin_public, 'init');
+		$this->loader->add_action( 'wp_enqueue_scripts', $plugin_public, 'enqueue_styles' );
+		$this->loader->add_action( 'wp_enqueue_scripts', $plugin_public, 'enqueue_scripts' );
+		$this->loader->add_action( 'init', $plugin_public, 'init' );
 
 		// Register dynamic hook
-		$hook_name = get_option('tonicpow_hook_name');
-		switch ($hook_name[0]) {
+		$hook_name = get_option( 'tonicpow_hook_name' );
+		switch ( $hook_name[0] ) {
 			case "media_upoload_video":
```

Comment:
@Abydars do you know if this is a real hook or a typo?

`media_upoload_video` vs `media_upload_video` ?